Sources/Web3Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -216,6 +216,54 @@ public class BIP32Keystore: AbstractKeystore {
         try encryptDataToStorage(password, data: serializedRootNode, aesMode: self.keystoreParams!.crypto.cipher)
     }
 
+    /// Fast generation addresses for current account
+    /// used for shows which address user will get when changed number of his wallet
```

Comment:
Edit suggestion:
```
/// used to show which addresses the user can get for indices from `0` to `number-1`
```

Reply:
my english is not very good((

Reply:
That's okay. Mine wasn't good either. Sometimes still isn't good 🙂